src/static-apps.js

```diff
@@ -1,6 +1,18 @@
 import React from 'react'
 import { IconSettings, IconPermissions, IconApps } from '@aragon/ui'
-import IconHome from './icons/app-home.svg'
+import IconHomeSvg from './icons/app-home.svg'
+
+const IconHome = () => (
```

Comment:
Maybe we should place this in the icons directory?

Reply:
I went a bit farther and refreshed a bit the way we manage app icons in general: https://github.com/aragon/aragon/pull/655/commits/edfc2cfae5a0a7fb62e10989fd295d645d32bdf5